### ISSUE TITLE
My Cyclescape

### ISSUE THREAD

**Issue by si-the-pie** [cyclestreets/cyclescape#773]
The 'My Cyclescape' section of the site is where I'd expect to go to quickly see the items of most interest to me.

Currently this view pre-selects 'My subscribed threads' and that often contains items that I probably ought to care about and read but they not really my main focus.

What I really want is a view of all 'My recent posts' so that I can quickly identify what I last posted upon and find out 'what is happening'.

Basically what I'd like is a preview list of all my postings, with and indication of whether there have been any further postings on that thread since my last contribution there.

That list could be sorted either by my last posting date, or by the date of the latest post.

**Comment by mvl22** [comment]
This will be mopped up during redesign.
